daemon/firewall/firewall_manager.go

```diff
@@ -0,0 +1,333 @@
+package firewall
```

Comment:
`iptables` subpackage could be removed eventually.`firewall` will be greatly simplified, we'll have `firewall_manager` and maybe `allowlist_routing`.

Reply:
Created a sub package to avoid circular dependency shenanigans. Will move it to `iptables`(or just rename the new package) once we can get rid of the old stuff and restructure.